app/vue/src/server/config/webpack.config.js

```diff
@@ -52,7 +52,7 @@ export default function() {
       // Based on this CRA feature: https://github.com/facebookincubator/create-react-app/issues/253
       modules: ['node_modules'].concat(nodePaths),
       alias: {
-        'vue$': 'vue/dist/vue.esm.js'
+        'vue': path.resolve(path.join(__dirname, '../../../node_modules', 'vue/dist/vue.esm.js'))
```

Comment:
Not correct in our case because of webpack being in storybook/vue and not ine the example app. 

Not pointing to the correct dep will have webpack load 2 times vie and thus make it impossible to ise extension or global custom components